src/goto-programs/remove_asm.cpp

```diff
@@ -305,24 +298,24 @@ void remove_asmt::process_function(
 }
 
 /// removes assembler
-void remove_asmt::operator()()
+void remove_asm(
+  goto_functionst::goto_functiont &goto_function,
```

Comment:
I don't see `goto_function.first` being used anywhere here.

Reply:
I don't understand the comment. This argument is just passed to `remove_asmt::process_function`, which Daniel wrote.

Reply:
I'd propose to leave this to #1681.